recipes/async_generator/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "async_generator" %}
+{% set version = "1.9" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "b7d5465c6174fe86dba498ececb175f93a6097ffb7cc91946405e1f05b848371" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - async_generator
+    - async_generator._tests
+
+about:
+  home: https://github.com/python-trio/async_generator
+  license: MIT
+  license_family: MIT
+  # license file not available on sdist yet: https://github.com/python-trio/async_generator/pull/11
+  license_file: ''
```

Comment:
You don't need the empty field but that does not hurt.

Reply:
I prefer to leave it there so we fill it with the license file in the next release